source/config/__init__.py

```diff
@@ -1304,6 +1305,56 @@ def __setitem__(
 		self.manager._markWriteProfileDirty()
 		self._cache[key] = val
 
+		# Alias [documentFormatting][reportFontAttributes] for backwards compatibility.
+		# Todo: Remove in 2025.1.
+		if version_year < 2025 and NVDAState._allowDeprecatedAPI():
+			self._linkDeprecatedValues(key, val)
+
+	def _linkDeprecatedValues(self, key: aggregatedSection._cacheKeyT, val: aggregatedSection._cacheValueT):
```

Comment:
I think it makes sense to keep the structure of this method in 2025.1, making it a no-op. That prevents us from reinventing the wheel if we have to build a similar structure in 2025.2.
In that case, that probably needs to be added in a comment or in the doc string

Reply:
My thought was to add sample code based on this code to the developer docs. Function calls are expensive, and I don't particularly like the idea of having a function call that explicitly never does anything. I suppose we could comment out the call to `_linkDeprecatedValues`, until it's later needed...

Reply:
AH, both make sense to me. As long as this code doesn't get lost, it's fine.

Reply:
I think it would best to just keep the function here, comment out the current case as an example.